### DISCUSSION TITLE
Scraping sites that require a login

### DISCUSSION THREAD

**Discussion by lcpizani** [ScrapeGraphAI/Scrapegraph-ai#401] [Q&A]
Hey, very nice web scraping tool, congratulations @VinciGit00!

Sometimes, I cannot use the SmartScraperGraph on certain websites that require a login. In a previous discussion, you said it was possible to make the authentication ourselves and pass it as a string parameter in the graph. How can I do that?

Thank you!

**Comment by VinciGit00** [comment]
hi, we do not natively support the login, you should make the login by yourself for example using the request library and make a script similar to this one [https://github.com/VinciGit00/Scrapegraph-ai/blob/main/examples/benchmarks/SmartScraper/benchmark_groq.py](url)

**Reply by lcpizani** [reply]
Thank you. 
Could you please send the link to the script again? For me it Is not working

**Reply by dyh2024** [reply]
> hi, we do not natively support the login, you should make the login by yourself for example using the request library and make a script similar to this one [https://github.com/VinciGit00/Scrapegraph-ai/blob/main/examples/benchmarks/SmartScraper/benchmark_groq.py](url)

Could you please send the link to the script again? the link is 404

**Comment by VinciGit00** [comment]
please update to the new version
